api/tests/identity-access-management/integration/domain/usecases/update-user-details-by-admin.usecase.test.js

```diff
@@ -6,15 +7,26 @@ import {
   AlreadyRegisteredEmailError,
   AlreadyRegisteredUsernameError,
 } from '../../../../../src/shared/domain/errors.js';
-import { catchErr, databaseBuilder, expect } from '../../../../test-helper.js';
+import { catchErr, databaseBuilder, expect, sinon } from '../../../../test-helper.js';
 
 describe('Integration | Identity Access Management | Domain | UseCase | updateUserDetailsByAdmin', function () {
   let userId;
+  let updatedByAdminId;
+
+  let clock;
+  const now = new Date('2024-12-25');
 
   beforeEach(async function () {
     userId = databaseBuilder.factory.buildUser({ email: 'email@example.net' }).id;
+    updatedByAdminId = databaseBuilder.factory.buildUser({ email: 'admin@example.net' }).id;
```

Comment:
Une suggestion pas du tout obligatoire mais qui permet de nous rapprocher de l'utilisation réelle : 

```suggestion
    updatedByAdminId = databaseBuilder.factory.buildUser.withRole({
          email: 'admin@example.net'
          role: ROLES.SUPER_ADMIN,
        }).id;
```